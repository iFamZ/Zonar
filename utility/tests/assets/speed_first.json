{
    "type": "speed first then 2 out of order data then correct lat long",
    "data": [
        {
            "ts": 1650500050,
            "k": "speed",
            "v": "55.75"
        },
        {
            "ts": 1650500049,
            "k": "longitude",
            "v": "-122.335167"
        },
        {
            "ts": 1650500048,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500051,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500051,
            "k": "longitude",
            "v": "-122.335160"
        }
    ]
}