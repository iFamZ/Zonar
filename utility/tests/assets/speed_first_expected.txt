Out of order detected
Out of order detected
{'latitude': '47.608013', 'longitude': '-122.335160', 'speed': '55.75', 'ts': 1650500051}
